scripts/deb/DEBIAN/postinst

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# install desktop launcher
+xdg-desktop-menu install /opt/translationcore/unfoldingword-translationcore.desktop
+
+# install icon
+xdg-icon-resource install --size 128 /opt/translationcore/translationcore-128.png unfoldingword-translationcore
```

Comment:
@RoyalSix You have a bunch of new files that make me think you branch off of develop branch. is this correct?